symbology/custom-dictionary-style/README.md

```diff
@@ -10,27 +10,33 @@ When symbolizing geoelements in your map, you may need to convey several pieces
 
 ## How to use the sample
 
-Pan and zoom the map to see the symbology from the custom dictionary style.
+Use the radio buttons to toggle between the dictionary symbols from the web style and style file. Pan and zoom around the map to see the symbology from the chosen dictionary symbol style.
 
 ## How it works
 
-1. Create a new `DictionarySymbolStyle` by passing in the path to the custom style file (.stylx).
-2. Create a new `DictionaryRenderer`, providing the dictionary symbol style.
-3. Apply the dictionary renderer to a feature layer using `layer.setRenderer(dictionaryRenderer)`.
+1. Create a `PortalItem`, referring to a `Portal` and the item ID of the web style.
+2. If the web style toggle has been selected, create a new `DictionarySymbolStyle` from the portal item using `new DictionarySymbolStyle(portalItem)`
+3. If the file style toggle has been selected, create a new dictionary symbol style using `DictionarySymbolStyle.createFromFile(stylxFile.getAbsolutePath())`
```

Comment:
```suggestion
2. Based on the style selected:
  -  If the web style toggle has been selected, create a new `DictionarySymbolStyle` from the portal item using `new DictionarySymbolStyle(portalItem)`
  -  If the file style toggle has been selected, create a new dictionary symbol style using `DictionarySymbolStyle.createFromFile(stylxFile.getAbsolutePath())`
```

Just stepping out of the numbered ordering here since they're "ifs"

Reply:
Agree with this. I also think we could perhaps incoroprate point 1 (creating the PortalItem` into the respective 'if' here, as the portal item is only necessary for that particular action, not for the whole sample.